go.mod

```diff
@@ -3,10 +3,12 @@ module github.com/launchdarkly/ld-find-code-refs
 go 1.13
 
 require (
+	github.com/antihax/optional v1.0.0
 	github.com/go-git/go-git/v5 v5.1.0
+	github.com/golang/protobuf v1.4.3 // indirect
 	github.com/hashicorp/go-retryablehttp v0.5.0
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
-	github.com/launchdarkly/api-client-go v0.0.0-20181031211650-fe4f1d017b6e
+	github.com/launchdarkly/api-client-go v3.9.0+incompatible
```

Comment:
Updated api-client-go to the latest version to bring in support for the `archived` filter.